plugin/pkg/scheduler/algorithm/well_known_labels.go

```diff
@@ -28,6 +28,26 @@ const (
 	// and removed when node becomes reachable (NodeReady status ConditionTrue).
 	TaintNodeUnreachable = "node.alpha.kubernetes.io/unreachable"
 
+	// When feature-gate for TaintBasedEvictions=true flag is enabled,
+	// TaintNodeOutOfDisk would be automatically added by node controller
+	// when node becomes out of disk, and removed when node has enough disk.
+	TaintNodeOutOfDisk = "node.alpha.kubernetes.io/outOfDisk"
```

Comment:
What is the rationale behind including alpha in the taint name? Is there concern a taint named `node.kubernetes.io/outOfDisk` would want to be used in some other way in the future? If not, I'd prefer taking the approach recommended for API fields:

1. Carefully choose names
2. Feature gate the setting of the data while in alpha
3. On graduation to beta, the feature gate can be enabled by default

That means that if the feature survives, clusters that enabled and used the taints in alpha state can simply continue to do so.

cc @kubernetes/sig-api-machinery-pr-reviews